OWNERS

```diff
@@ -5,3 +5,4 @@ approvers:
   - lburgazzoli     # Luca Burgazzoli <lburgazzoli@gmail.com>
   - xiang90         # Xiang Li <xiang.li@coreos.com>
   - heyitsanthony   # Anthony Romano <anthony.romano@coreos.com>
+  - vorburger       # Michael Vorburger <mike@vorburger.ch>
```

Comment:
Hey team - We need to make sure https://github.com/kubernetes/org/blob/main/config/etcd-io/sig-etcd/teams.yaml#L79-L84 aligns with this file and vice versa.

Looks like it's time to refresh/update both.

Reviewing recent commits and https://github.com/etcd-io/jetcd/graphs/contributors it looks to me like this file as well as the counterpart in `k/org` should be updated to list just @lburgazzoli and @vorburger as the active OWNERS.

Do folks agree with that?  If so we can update this pr to reflect and then update `k/org` following.

Reply:
@jmhbnz and @jberkus I'm sure it's up to me to remove @fanminshi (who IS on the aforementioned file, and [the 2nd top contributor](https://github.com/etcd-io/jetcd/graphs/contributors); I'd suggest what we definitely keep at least him here as 3rd contrib) or @xiang90 or @heyitsanthony (who are not). Folks, seeing that you have apparently never actually contributed to this repo, I've taken the liberty of removing you.

I've fixed the DCO with (stupid) `git commit --amend -s` as well.

@jmhbnz @jberkus @lburgazzoli merge this as-is now?